sirepo/template/radia.py

```diff
@@ -43,11 +92,11 @@
 _SIM_DATA, SIM_TYPE, _SCHEMA = sirepo.sim_data.template_globals()
 _SDDS_INDEX = 0
 
+#DATA_REQUESTER = DataRequester()
```

Comment:
What's the purpose of DataRequester if it isn't used?

Reply:
Just a placeholder while the export stuff gets filled out. Also to see if it's the right way to go, like should it even exist...